content/synthetics/_index.md

```diff
@@ -2,19 +2,61 @@
 title: Synthetics
 kind: documentation
 beta: true
-description: Synthetics
+description: documentation
 aliases:
   - /integrations/synthetics/
 further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
   tag: "Blog"
   text: "Introducing Datadog Synthetics"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
 ---
 
-<div class="alert alert-warning">Synthetics is currently in beta. See the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>
+<div class="alert alert-warning">Synthetics is currently in beta for the Datadog US Region. To request access complete the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>
 
-Datadog is pleased to announce the upcoming availability of Synthetics, a whole new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
+## Overview
+
+Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+## Search 
+
+Advanced search lets you query checks by any combination of checks attributes:
+
+* `title` and `message` - text search
+* `status` - Alert, No Data, Ok
+* `creator`
+* `region`
+* `muted`
+* `notification` 
+* `tags`
+
+To run a search, construct your query using the checkboxes on the left and/or the search bar along the top. The search bar updates with the equivalent query as you check the boxes. Likewise, when you modify the search bar query (or write one from scratch), the checkboxes update to reflect that change. Query results update in real-time as you edit the query; there's no 'Search' button to click.
```

Comment:
This whole sentence is quite hard to understand. Would it be possible to have a gif alongside to illustrate the facetted search?

Reply:
Facetted search is explained in many parts of the documentation, let's not overload the page here with a gif yet since design will soon change. When new design will be out we will add a small gif indeed :)